.changelog/3755.bugfix.2.md

```diff
@@ -0,0 +1 @@
+go/oasis-node: Dump correct block height on halt
```

Comment:
I think this Change Log fragment should link to this PR or the PR where this goes into the `master` branch.

Reply:
Good idea, changed to reference this PR.